docs/basic-features/environment-variables.md

```diff
@@ -110,7 +110,7 @@ Next.js allows you to set defaults in `.env` (all environments), `.env.developme
 
 `.env.local` always overrides the defaults set.
 
-> **Note**: `.env`, `.env.development`, and `.env.production` files should be included in your repository as they define defaults. **`.env.*.local` should be added to `.gitignore`**, as those files are intended to be ignored. `.env.local` is where secrets can be stored.
+> **Note**: `.env`, `.env.development`, and `.env.production` files should be included in your repository as they define defaults. **`.env*.local` should be added to `.gitignore`**, as those files are intended to be ignored. `.env.local` is where secrets can be stored.
```

Comment:
```suggestion
> **Note**: `.env`, `.env.development`, and `.env.production` files should be included in your repository as they define defaults. **`.env.local` should be added to `.gitignore`**, as those files are intended to be ignored. `.env.local` is where secrets can be stored.
```

Why not `.env.local` which is the name of the file described above?

Reply:
Sorry for the late reply!  
As original document said `.env.*.local`, I had guessed some developer uses multiple `.env.local` for example `.env.alpha.local` or `.env.beta.local`.  
However I agree to your suggestion because this document does not need to support such a complex situation.